template/build/webpack.base.conf.js

```diff
@@ -92,5 +93,10 @@ module.exports = {
     net: 'empty',
     tls: 'empty',
     child_process: 'empty'
-  }
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env': utils.stringifiedEnvVars
```

Comment:
a little nitpicky, but this fn name makes me think that ALL env vars will be stringified and exposed.